src/test/shell/bazel/BUILD

```diff
@@ -236,6 +236,56 @@ sh_test(
     ],
 )
 
+sh_test(
```

Comment:
opt: I'd consider using a list comprehension to generate these rules so as to have as little duplication as possible

Reply:
Done.